pkg/provider/aws/plugin.go

```diff
@@ -109,6 +109,16 @@ func (a *AWS) createEksClusters(result *core.ConstructGraph, dag *core.ResourceG
 	}
 
 	vpc := resources.CreateNetwork(a.Config, dag)
+	sg := resources.GetSecurityGroup(a.Config, dag)
+	sg.IngressRules = append(sg.IngressRules, resources.SecurityGroupRule{
+		Description: "Allows ingress traffic from the EKS control plane",
+		FromPort:    9443,
+		Protocol:    "TCP",
+		ToPort:      9443,
+		CidrBlocks: []core.IaCValue{
+			{Property: "0.0.0.0/0"},
```

Comment:
Do we want this to be configurable? or are we okay with...

a **hard**<sup>coded</sup> **cidr**?

Reply:
this is a specific rule for the eks control plane and im not sure if it can be changed ever for the control plane to work so hardcoded should be fine. We maybe can look into if theres a narrower cidr but i didnt think they provided one

Reply:
I was just making a pun

Reply:
oh i see it now lol